include/IECore/Timer.h

```diff
@@ -92,13 +93,25 @@ class IECORE_API Timer
 
 class IECORE_API ScopedTimer
 {
+	typedef std::string Channel;
+
 	public:
-		ScopedTimer( const std::string &name );
+		ScopedTimer( const std::string &name, const Channel &channel = "" );
```

Comment:
In particular I'd be interested in the relationship between `name` and `channel`, since it seems like they're kindof similar concepts? Why is one "string label thingy" insufficient?

Reply:
Would you guys prefer a bool to switch from a ScopedTimer that prints the result on destruction to one that just adds it to a "channel"? To avoid the duplicated label thingy?

Reply:
That's one of the things that occurred to me when I was trying to understand this. The other was whether it might be an option to have a separate ChannelTimer and leave the ScopedTimer alone? Also, it seems that ScopedTimer doesn't offer the same choice of `Timer::Mode` that Timer does...